postr/youtube_postr.py

```diff
@@ -91,6 +96,27 @@ def get_user_likes(self) -> int:
             mine='true',
         )['items'][0]['statistics']['subscriberCount'])
 
+    def get_user_videos(self) -> List[str]:
+        ''' This method returns the video ids of this user.'''
+        # Returns a list of video ids where the first item is the most recently uploaded video
+        upload_id = channels_list_by_id(
+            self.build,
+            part='contentDetails',
+            id=self.channel_id,
+        )['items'][0]['contentDetails']['relatedPlaylists']['uploads']
+
+        upload_playlist_json = playlist_items_list_by_playlist_id(
+            self.build,
+            part='contentDetails',
+            maxResults=50,
+            playlistId=upload_id,
+        )
+
+        upload_id_list = []
+        for uploaded_video in upload_playlist_json['items']:
+            upload_id_list.append(uploaded_video['contentDetails']['videoID'])
```

Comment:
replace these lines with

`return [video['contentDetails']['videoID'] for video in upload_playlist_json['items']]`

can multi line but it's pretty short with good names

Reply:
wow you never made it nicer :(